src/lang-promql/client/prometheus.ts

```diff
@@ -200,8 +200,8 @@ export class HTTPPrometheusClient implements PrometheusClient {
     return this.labelValues('__name__');
   }
 
-  private fetchAPI<T>(resource: string): Promise<T> {
-    return this.fetchFn(this.url + resource)
+  private fetchAPI<T>(input: string, init?: RequestInit): Promise<T> {
```

Comment:
`resource` was good for me. I don't see the reason to change it to `input`. It's not really clearer from my point of view.

Reply:
![image](https://user-images.githubusercontent.com/3659110/116001997-110f8880-a62a-11eb-9b6f-1dfe7212cce6.png)
I just saw that the original parameter name for the fetch api was `input`, so I changed it to `input` here. if you like the name `resource`, we can keep it.